.infra/index.ts

```diff
@@ -306,6 +306,10 @@ if (isAdhocEnv) {
         { targetPort: 3000, port: 80, name: 'http' },
         { targetPort: 9464, port: 9464, name: 'metrics' },
       ],
+      spot: {
+        enabled: true,
+        weight: 50,
+      },
```

Comment:
Looks good on 50% as a start, we just need to be carful on evictions so load is handled throughout the day.

Reply:
Yeah, my thoughts too.
I might want to try at one point with two deployments of api, one that's always 3 replicas with no auto scaling to keep a minimum, on regular nodes. Then a 2nd deployment that's "only" scheduled on spot nodes, that can scale to 0

Reply:
Our current spot nodes that host the redis replicas, has an age of 30 days. So evictions doesn't happen too often. But it can happen
<img width="468" alt="image" src="https://github.com/user-attachments/assets/1595d0e8-daae-417f-a193-0be20613e12a">